packages/atlas/src/api/hooks/killSwitch.ts

```diff
@@ -0,0 +1,30 @@
+import { MutationHookOptions, QueryHookOptions } from '@apollo/client'
+
+import {
+  GetKillSwitchQuery,
+  GetKillSwitchQueryVariables,
+  SetKillSwitchMutation,
+  SetKillSwitchMutationVariables,
+  useGetKillSwitchQuery,
+  useSetKillSwitchMutation,
+} from '@/api/queries'
+
+export const useSetKillSwitch = (opts?: MutationHookOptions<SetKillSwitchMutation, SetKillSwitchMutationVariables>) => {
+  const [setKillSwitch, rest] = useSetKillSwitchMutation()
+  return {
+    setKillSwitch: (isKilled: boolean, secret: string) =>
+      setKillSwitch({
+        variables: { isKilled },
+        context: { headers: { authorization: secret }, ...opts },
+      }),
+    ...rest,
+  }
+}
+
+export const useGetKillSwitch = (opts?: QueryHookOptions<GetKillSwitchQuery, GetKillSwitchQueryVariables>) => {
+  const { data, ...rest } = useGetKillSwitchQuery(opts)
```

Comment:
Let's add some persistence to this data. So that if kill switch was enabled on the last app visit, I will see the maintenance screen by default and not flash of the app loading:
```tsx
const { isLoading, isKilled, wasKilledLastTime } = useMagic()

if (isKilled || (isLoading && wasKilledLastTime)) return <Maintenance />
else return <RegularStuff />
```